apps/site/components/EOL/EOLModal/index.tsx

```diff
@@ -0,0 +1,66 @@
+import { Modal, Title, Content } from '@node-core/ui-components/Common/Modal';
+import { useTranslations } from 'next-intl';
+import { useMemo } from 'react';
+import type { FC, ComponentProps } from 'react';
+
+import KnownSeveritySection from '#site/components/EOL/KnownSeveritySection';
+import UnknownSeveritySection from '#site/components/EOL/UnknownSeveritySection';
+import { SEVERITY_ORDER } from '#site/next.constants.mjs';
+import type { NodeRelease } from '#site/types/releases';
+import type { Vulnerability } from '#site/types/vulnerabilities';
+
+type EOLModalProps = ComponentProps<typeof Modal> & {
+  release: NodeRelease;
+  vulnerabilities: Array<Vulnerability>;
+};
+
+const EOLModal: FC<EOLModalProps> = ({
+  release,
```

Comment:
Could we destruct this to:


```suggestion
  { major, codename },
```

Reply:
addressed via https://github.com/nodejs/nodejs.org/pull/7990/commits/ef51f5cc7551c65365781c3d08d33822341de846